### PR TITLE
Added a developer

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@samyakbvs PLease share preview link

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyak-fossasia.org/

**Review by kubami9 — APPROVED** [review]
Works fine

**Review by ParthS007 — APPROVED** [review]

**Review by YashKumarVerma — APPROVED** [review]

**Comment by vaibhavsingh97** [comment]
Just a small question 😄 Why you change the developer profile 😅

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM

**Comment by anshumanv** [comment]
Uneven rows
image

**Comment by anshumanv** [comment]
Also please check, there is no `Alumni` section in your link.

**Comment by vaibhavsingh97** [comment]
@anshumanv Thanks for pointing it out

**Review by anshumanv — CHANGES_REQUESTED** [review]
Check comments.

**Comment by ParthS007** [comment]
Thanks @anshumanv for pointing this @samyakbvs Please correct this 👍

**Review by isuruAb — COMMENTED** [review]
@samyakbvs  please check the rows problem. Add Areeb's photo at the end of the list.

**Comment by samyakbvs** [comment]
Updated @isuruAb and @anshumanv the code is same as the repo and if you still doubt me then please the the file changed section which clearly show my changes

**Comment by samyakbvs** [comment]
The alumni code is missing from the repo and I don't think that's my fault?

**Comment by anshumanv** [comment]
Alumni section is present on the page, please see.

**Comment by samyakbvs** [comment]
But not in the repo

**Comment by samyakbvs** [comment]
the code is from the fossasia github repo

**Comment by samyakbvs** [comment]
@anshumanv

**Comment by samyakbvs** [comment]
Is it ready to merge then?

**Comment by anshumanv** [comment]
No, unless those errors are solved, try to find a workaround, read the code well, don't be in a haste to get your PR merged and task approved, the issue is for you to figure out, solve and code.

**Comment by samyakbvs** [comment]
But why isnt the code in the repo where will i get it from?

**Comment by anshumanv** [comment]
The code is in the repo :wink: sections won't appear in a website magically.
Please see.

**Comment by samyakbvs** [comment]
Okay I will

**Comment by samyakbvs** [comment]
Any other issue?

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Why you removed a developer from there

**Comment by anshumanv** [comment]
> Any other issue?

Alumni section and improper rows, solve these two please. :smile:

**Comment by samyakbvs** [comment]
Okay

**Comment by samyakbvs** [comment]
Any tips on how to solve them?

**Comment by samyakbvs** [comment]
I thought that will fix the column issue but it didn't

**Comment by samyakbvs** [comment]
@ms10398

File: team.html [hunk 1]
```diff
@@ -1467,7 +1485,577 @@ <h1>Enablement Team</h1>
...
+<h1>Alumni</h1>
+</div>
+</div>
+<div class="col-md-3 col-sm-6 speaker-column">
+<div class="speaker">
+<div class="image-holder">
+<img class="background-image" alt="Arnav Gupta" src="img/ArnavGupta.jpg">
+<div class="hover-state text-center preserve3d">
+<div class="social-links vertical-align">
+<a href="https://github.com/championswimmer"><i class="icon fa fa-github"></i></a>
```
Comment (abishekvashok) [inline team.html]: Please properly intend the code :)

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
Can someone pls guide me on how to fix the column issue

**Comment by samyakbvs** [comment]
@abishekvashok

**Comment by samyakbvs** [comment]
Finally done @anshumanv

**Comment by samyakbvs** [comment]
@abishekvashok how would i be able to indent suck a long code?

**Comment by samyakbvs** [comment]
@ms10398

**Comment by samyakbvs** [comment]
Please approve without indentation

**Comment by NaviRocker** [comment]
Try to Squash your Commits

**Review by ParthS007 — APPROVED** [review]

**Comment by samyakbvs** [comment]
something went wrong creating a new pull request

**Review by 0x48piraj — COMMENTED** [review]
Messed up!

**Pull request closed without merging**